webportal/src/app/env.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+window.ENV = {
```

Comment:
I'm curious about how your method works. I guess this env.js is a template, and it will be dynamically instantiated by envsub inside the container before the service starts (the 'prestart' command of npm). I'm I right?

Reply:
@hwuu You are right, this makes the environment variables injected to window.ENV in the runtime, not related to webpack build.